core/src/main/scala/tech/beshu/ror/api/TestConfigApi.scala

```diff
@@ -126,19 +133,21 @@ class TestConfigApi(rorInstance: RorInstance)
   }
 
   private def forceReloadTestConfig(config: RawRorConfig,
-                                    ttl: FiniteDuration)
+                                    ttl: FiniteDuration Refined Positive)
                                    (implicit requestId: RequestId): EitherT[Task, TestConfigResponse, Unit] = {
     EitherT(
       rorInstance
         .forceReloadTestConfigEngine(config, ttl)
         .map {
           _.leftMap {
-            case RawConfigReloadError.ReloadingFailed(failure) =>
-              TestConfigResponse.UpdateTestConfig.FailedResponse(s"Cannot reload new settings: ${failure.message}")
-            case RawConfigReloadError.ConfigUpToDate(_) =>
+            case IndexConfigSavingError(error) =>
+              TestConfigResponse.UpdateTestConfig.FailedResponse(s"Cannot save new settings: ${error.show}")
```

Comment:
the method is to force reload the config, but we inform the api user that settings cannot be saved. Sth is wrong here